src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-__version__ = "2.0.7+dev"
```

Comment:
This should be 2.0.8+dev.
The +dev is still needed in the repo but the release bot will remove this on release.